specification/paths/Returns-v1-return-methods.json

```diff
@@ -129,9 +129,13 @@
                       },
                       "attributes": {
                         "required": [
-                          "return_type",
-                          "default_weight"
-                        ]
+                          "return_type"
+                        ],
+                        "properties": {
+                          "default_weight": {
+                            "description": "Required if the `return_type` is set to `shipment`."
+                          }
+                        }
                       },
                       "relationships": {
```

Comment:
the relationships `contract` and `service` are also required if the `return_type = shipment`. could you add a similar comment to those properties to indicate this to the user?